### DISCUSSION TITLE
Possible to disable the animations of the indent plugin?

### DISCUSSION THREAD

**Discussion by benfrain** [folke/snacks.nvim#255] [Q&A] [answered]
Just busy trying Snacks to replace a bunch of separate plugins.

I'd like to disable the animation that occurs on the indent lines. Is this possible? I tried using an option the same as scope but this doesn't work:

```lua
indent = { enabled = true, char = "▎", only_scope = true, only_current = true, animate = { enabled = false } },
```

**Comment by folke** [comment, marked as answer]
```lua
      indent = {
        scope = {
          animate = { enabled = false },
        },
      },
```

If that doesn't work for you, then you're either overriding the config somewhere else, or not calling setup (or use opts when using lazy.nvim)

**Reply by benfrain** [reply]
My bad, I was looking at https://github.com/folke/snacks.nvim/blob/main/docs/indent.md and thought that the options for scope were at the same 'level' as indent so had them like alongside each other like this:

```lua
indent = { enabled = true, char = "▎", only_scope = true, only_current = true },
scope = { enabled = false, animate = { enabled = false } },
```

**Reply by davidosomething** [reply]
^ i thought so too

**Reply by folke** [reply]
That is also the case, but all the indent options need to be put under the `opts.indent` section of snacks, since they're for the `indent` plugin. The readme for indent (and the others) shows that right above the config section.
